### PR TITLE
PR269: Replacing all inputs with PrevPointInput

### PR DESCRIPTION
**Pull request by crsmyz**
Replacing the remaining MUI Input components with the PrevPointInput.

### CODE REVIEW

File: frontend/src/components/LoginForm.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ import "../scss/login-form.scss"
 import FormGroup from "@material-ui/core/FormGroup"
 import FormControl from "@material-ui/core/FormControl"
 import InputLabel from "@material-ui/core/InputLabel"
-import Input from "@material-ui/core/Input"
+// import Input from "@material-ui/core/Input"
```
Comment (ThrowsException) [inline frontend/src/components/LoginForm.js:11]: delete

File: frontend/src/components/UserSearch.js [hunk 2]
```diff
@@ -4,7 +4,8 @@ import "../scss/participant-search.scss"
 import FormGroup from "@material-ui/core/FormGroup"
 import FormControl from "@material-ui/core/FormControl"
 import InputLabel from "@material-ui/core/InputLabel"
-import Input from "@material-ui/core/Input"
+// import Input from "@material-ui/core/Input"
```
Comment (ThrowsException) [inline frontend/src/components/UserSearch.js:7]: delete

File: frontend/src/components/LoginForm.js (hunk 1 above)
Comment (crsmyz) [reply frontend/src/components/LoginForm.js:11]: Left over from testing,deleting now.

**Comment by crsmyz** [comment]
> Are the ids used for anything? i think they are redundant



> Are the ids used for anything? i think they are redundant



> Are the ids used for anything? i think they are redundant

Yea, doesn't look like they're in use. I'll remove them

**Pull request merged**